src/setup.js

```diff
@@ -0,0 +1,8 @@
+import 'dotenv/config';
+import { client } from './utils/db.js';
+// eslint-disable-next-line no-unused-vars
+import { User } from './models/user.js';
+// eslint-disable-next-line no-unused-vars
+import { Token } from './models/token.js';
+
+client.sync({ force: true });
```

Comment:
Be cautious when using `force: true` with `client.sync()`. This will drop and recreate all tables, which can lead to data loss. Ensure this is the intended behavior, especially in a production environment.